docs/devel/release.md

```diff
@@ -0,0 +1,64 @@
+# runtime-tools release guide
+
+## Release process
+
+This section shows how to perform a release of runtime-tools.
+The following example assumes we're going from version 0.5.0 (`v0.5.0`) to 0.6.0 (`v0.6.0`).
+
+Let's get started:
+
+- Start at the relevant milestone on GitHub (e.g. https://github.com/opencontainers/runtime-tools/milestones/v0.6.0): ensure all referenced issues are closed (or moved elsewhere, if they're not done). Close the milestone.
+- runtime-tools does not use a [roadmap file](https://github.com/opencontainers/runtime-tools/issues/465) but GitHub milestones. Update the [other milestones](https://github.com/opencontainers/runtime-tools/milestones), if necessary
+- Branch from the latest master, make sure your git status is clean
+- Update the [VERSION](https://github.com/opencontainers/runtime-tools/blob/master/VERSION)
+- Update the [release notes][changelog].
+  Try to capture most of the salient changes since the last release, but don't go into unnecessary detail (better to link/reference the documentation wherever possible).
+
+Ensure the branch is correct:
+
+- Ensure the build is clean!
+  - `git clean -ffdx && make && make test` should work
```

Comment:
Which OS/arch(es) are we building for?  Just `GOOS=linux GOARCH=amd64`?  Do we want to build for each of the OSes listed [here][1]?  For some set of arches?

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/spec.md#platforms

Reply:
Ideally, all, but I don't know how to do that. Could we start with linux + amd64 in this PR and improve later based on contributions? But we should at least namespace the filename of the tarball correctly.

Reply:
> Could we start with linux + amd64 in this PR and improve later based on contributions?

That's fine with me, but ideally these instructions are the same for all platforms.  I think the missing piece is a build farm supporting all of our target platforms.  Are we waiting for Windows and Solaris folks to point us at farms for their platforms?